packaging/debian/copyright

```diff
@@ -0,0 +1,24 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: libsxg
+Source: https://github.com/google/libsxg
+
+Files: *
+Copyright: 2019 Google LLC <kumagi@google.com>
```

Comment:
No need of email address.
https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#files-paragraph

Reply:
Removed.